### PR TITLE
docs: update cli reference

### PR DESCRIPTION
**Pull request by ldelossa**
sync up cli references

Signed-off-by: ldelossa <ldelossa@redhat.com>

### CODE REVIEW

File: Documentation/reference/config.md [hunk 1]
```diff
@@ -48,6 +52,18 @@ matcher:
...
+    sets: 
+      - alpine
+      - aws
+      - debian
+      - oracle
+      - photon
+      - pyupio
+      - rhel
+      - suse
+      - ubuntu
```
Comment (hdonnay) [inline Documentation/reference/config.md]:
```suggestion
    sets: null
```

File: Documentation/reference/config.md [hunk 2]
```diff
@@ -59,7 +75,8 @@ notifier:
     webhook: null
     amqp: null
     stomp: null
-auth: {}
+auth: 
+  psk: {}
```
Comment (hdonnay) [inline Documentation/reference/config.md]:
```suggestion
  psk: null
```

File: config/config.go [hunk 3]
```diff
@@ -50,14 +50,16 @@ type Config struct {
...
 	// "fatal"
 	// "panic"
-	LogLevel string   `yaml:"log_level" json:"log_level"`
-	Indexer  Indexer  `yaml:"indexer" json:"indexer"`
-	Matcher  Matcher  `yaml:"matcher" json:"matcher"`
+	LogLevel string  `yaml:"log_level" json:"log_level"`
+	Indexer  Indexer `yaml:"indexer" json:"indexer"`
+	Matcher  Matcher `yaml:"matcher" json:"matcher"`
+	// TODO(louis): this should not exist as a global config value
+	// and should exist inside the "Matcher" service config.
```
Comment (hdonnay) [inline config/config.go]:
```suggestion
```

We had a discussion when this got promoted out of the `Matcher` object

**Review by hdonnay — CHANGES_REQUESTED** [review]

File: Documentation/reference/config.md (hunk 2 above)
Comment (ldelossa) [reply Documentation/reference/config.md]: I'm using {} to signify a further object exists there

**Review by hdonnay — APPROVED** [review]

**Pull request merged**